tendrl/commons/utils/SSHUtil.py

```diff
@@ -0,0 +1,43 @@
+import psutil
+from tendrl.commons.event import Event
+from tendrl.commons.message import Message
+from tendrl.commons.utils import cmd_utils
+
+
+def sshd_status():
+    cmd = cmd_utils.Command("cat /var/run/sshd.pid")
```

Comment:
pid file is not guaranteed to exist. Furthermore, locations change. Since we're relying on systemd, a much more reliable method would be to use `systemctl show sshd.service` and grab the value of MainPID.